model-optimizer/extensions/back/ReverseInputChannels.py

```diff
@@ -75,7 +75,7 @@ def get_fw_index(node: Node, idx: int):
 
         assert len(order) > idx >= 0, \
             'Channel index {} is incompatible with old_api_map in node {}.'.format(idx, node_name)
-        return list(order).index(idx)
+        return int64_array(list(order).index(idx))
```

Comment:
it still seems to be of type numpy.ndarray

Reply:
Fixed.